metrics/ttfp/benchmark-ttfp.jl

```diff
@@ -0,0 +1,4 @@
+t_using = @elapsed using CairoMakie
+t_plot = @elapsed save("test.png", scatter(1:4))
```

Comment:
```suggestion
t_plot = (tstart = time(); save("test.png", scatter(1:4)); time() - tstart)
```
For latency it's safer to use this form because `@time` and `@elapsed` will precompile the workload. (If most of it is hidden behind runtime dispatch, this isn't a big deal, but in cases with really good inference you might eliminate the latency by trying to measure it.)